### PR TITLE
[299] Record which disabilities are disclosed by the trainee

### PR DESCRIPTION
**Pull request by darokel**
### Context

- https://trello.com/c/SOw5B0wN/299-m-record-which-disabilities-are-disclosed-by-the-trainee

### Changes proposed in this pull request

- Implements the disability selection section of the diversity flow

Screenshot 2020-11-03 at 15 43 33

Screenshot 2020-11-03 at 15 43 39

### Guidance to review

- Visit the review app - https://dfe-twd-rttd-pr-136.herokuapp.com/
- Create trainee and head to diversity information
- Fill out all details and choose to disclose disability, assert disabilities section is loaded with disability options
- Select a disability and assert your selection is saved when returning to this section

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -26,4 +26,4 @@ COPY . .
 
 RUN bundle exec rake assets:precompile
 
-CMD bundle exec rails db:migrate && bundle exec rails server -b 0.0.0.0
+CMD bundle exec rails db:migrate && db:seed && bundle exec rails server -b 0.0.0.0
```
Comment (vigneshmsft) [inline Dockerfile:29]: this `CMD` gets overridden during deployment
 
https://github.com/DFE-Digital/register-trainee-teachers/blob/master/terraform/modules/paas/variables.tf#L40

Comment (darokel) [reply Dockerfile]: I've reverted this and kept it out https://github.com/DFE-Digital/register-trainee-teachers/pull/136/commits/af428fb38391e196772f5f221ab5f6cdbeb25cef

**Comment by darokel** [comment]
[[M] Record which disabilities are disclosed by the trainee](https://trello.com/c/SOw5B0wN/299-m-record-which-disabilities-are-disclosed-by-the-trainee)

**Review by aliuk2012 — APPROVED** [review]

File: db/seeds.rb [hunk 2]
```diff
@@ -6,8 +6,12 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 
-# Load Nationalities
-
+# Load Nationalities. # TODO, move nationalities out of yaml.
```
Comment (defong) [inline db/seeds.rb:9]: Todo?

**Review by defong — APPROVED** [review]

File: spec/factories/disabilities.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
```
Comment (gpeng) [inline spec/factories/disabilities.rb:1]: Leave this for now but I wonder if we should use fixtures for all this seed type data. We don't and probably won't ever be adding functionality where we're mutating these things. We just need them to be around when the tests run.

**Review by gpeng — APPROVED** [review]

File: spec/factories/disabilities.rb (hunk 3 above)
Comment (darokel) [reply spec/factories/disabilities.rb:1]: So fixtures for simple models but factories where there's a bit more involved with setting up test data?

File: db/seeds.rb (hunk 2 above)
Comment (darokel) [reply db/seeds.rb:9]: A reminder for myself to move the nationalities into an initializer like the other stuff. I've created a ticket for it so i'll remove this comment

**Pull request merged**